services/web/server/src/simcore_service_webserver/users_api.py

```diff
@@ -105,6 +107,11 @@ async def is_user_guest(app: web.Application, user_id: int) -> bool:
 
 async def delete_user(app: web.Application, user_id: int) -> None:
     """Deletes a user from the database if the user exists"""
+    # FIXME: user cannot be deleted without deleting first all ist project
+    # otherwise this function will raise asyncpg.exceptions.ForeignKeyViolationError
+    # Consider "marking" users as deleted and havning a background job that
+    # cleans it up
```

Comment:
NOTE: and with the access rights.. if a project is shared and the owner is deleted we might end with some dangling projects that have no owner. I think since the prj_owner column is nullable that might work. but conceptually might need some more reflexions.

Reply:
I think this part needs of a deep review. I leave it like that here and create #1574